jobs/build/release/Jenkinsfile

```diff
@@ -249,14 +304,37 @@ node {
             }
 
             buildlib.registry_quay_dev_login()  // chances are, earlier auth has expired
+
             stage("mirror tools") {
-              retry(3) {
-                release.stagePublishClient(quay_url, dest_release_tag, release_name, arch, CLIENT_TYPE)
-              }
+                if (params.DIRECT_RELEASE_NIGHTLY) {
+                    return
```

Comment:
i think the signing will fail if we don't publish the client.

Reply:
Didn't look like much of a downside to allow the clients to be published for this. Removed short-circuit for the stage.